linkedevents/settings.py

```diff
@@ -440,11 +440,11 @@ def dummy_haystack_connection_for_lang(language_code):
 # like database and email that differ between development and production.
 f = os.path.join(BASE_DIR, "local_settings.py")
 if os.path.exists(f):
-    import imp
     import sys
+    import types
 
     module_name = "%s.local_settings" % ROOT_URLCONF.split(".")[0]
-    module = imp.new_module(module_name)
+    module = types.ModuleType(module_name)
```

Comment:
The documentation suggests using `importlib.util.module_from_spec` :thinking:

Reply:
It appears to be preferred. ModuleType documentation suggests 

> Use [importlib.util.module_from_spec()](https://docs.python.org/3/library/importlib.html#importlib.util.module_from_spec) to create a new module if you wish to set the various import-controlled attributes.

https://github.com/python/cpython/blob/main/Lib/imp.py#L49

` types.ModuleType(module_name)` appears to be identical to what the `imp.new_module` does